### PR TITLE
Add tests to avoid invalid code when chevrotainParserConfig is specified 

### LINKED ISSUE #513: Langium CLI generates invalid code when chevrotainParserConfig is specified for a specific language
When overwritting the chevrotainParserConfig property in the `langium-config.json`, Langium CLI generates an invalid `generate/module.ts` by adding the `<LanguageId>ParserConfig: IParserConfig` without importing IParserConfig.

image

## Error source
In [module-generator.ts](https://github.com/langium/langium/blob/main/packages/langium-cli/src/generator/module-generator.ts), the `IParserConfig` interface is imported only if `chevrotainParserConfig` is defined in the global configuration:
https://github.com/langium/langium/blob/82572206f172a91f6a09f3e69558cb96bb4392e5/packages/langium-cli/src/generator/module-generator.ts#L13
https://github.com/langium/langium/blob/82572206f172a91f6a09f3e69558cb96bb4392e5/packages/langium-cli/src/generator/module-generator.ts#L23
But never when it's defined in a specific language (and no in the global scope):
https://github.com/langium/langium/blob/82572206f172a91f6a09f3e69558cb96bb4392e5/packages/langium-cli/src/generator/module-generator.ts#L58-L59

Langium version: 0.3

## Steps To Reproduce

1. Generate the Hello World project
2. Modify the `langium-config.json` by adding `"chevrotainParserConfig": {"maxLookahead": 4},` in the language configuration.
Example:
```json
{
    "projectName": "SysmlV2",
    "languages": [{
        "chevrotainParserConfig": {"maxLookahead": 4},
        "id": "sysml-v2",
        "grammar": "src/language-server/sysml-v2.langium",
        "fileExtensions": [".sysml2"],
        "textMate": {
            "out": "syntaxes/sysml-v2.tmLanguage.json"
        }
    }],
    "out": "src/language-server/generated"
}
```
3. Run `npm run langium:generate` then `npm run build`

Minimal Langium project:
https://github.com/theogiraudet/Minimal-Langium-CLI-Error.git

## The current behavior
The absent of the import and a thrown exception.

## The expected behavior
The presence of the import.

### PR DESCRIPTION
**Pull request by Lotes**
Closes #513 . The actual problem was solved by @pluralia with #455 . I just added 2 tests to cover this scenario.

Please tell me if we can include these tests. They are pretty trivial. Looking up the required code in the generated file content string.

### CODE REVIEW

File: packages/langium-cli/jest.config.js [hunk 1]
```diff
@@ -7,7 +7,8 @@
 module.exports = {
     preset: 'ts-jest',
     roots: [
-        '<rootDir>/src'
+        '<rootDir>/src',
+        '<rootDir>/test',
```
Comment (msujew) [inline packages/langium-cli/jest.config.js:11]: Suggestion: We don't actually use this. Can you delete this file, while you're already at it?

File: packages/langium-cli/test/generator/module-generator.test.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            };
+            const grammarConfigMap = new Map<Grammar, LangiumLanguageConfig>();
+            grammarConfigMap.set(grammar, language);
+
+            // act
+            const moduleString = generateModule([grammar], config, grammarConfigMap);
+
+            // assert
+            expect(moduleString).toContain('MagicGrammarParserConfig');
+            expect(moduleString.split('\n').filter(line => line.startsWith('import') && line.indexOf('IParserConfig') > -1)).toHaveLength(1);
```
[the remark this answers is not in the document]
Comment (Lotes) [reply packages/langium-cli/test/generator/module-generator.test.ts]: Done, thanks.

File: packages/langium-cli/jest.config.js (hunk 1 above)
Comment (Lotes) [reply packages/langium-cli/jest.config.js]: Yes

**Review by msujew — APPROVED** [review]
LGTM, thanks!

**Pull request merged**